rskj-core/src/main/resources/expected.conf

```diff
@@ -0,0 +1,458 @@
+# disable wasabi+1 consensus rules by default for every network
```

Comment:
The idea is that the expected.conf file should contain all possible config settings which are expected by the node

Reply:
same as before, can you add this as a header comment on the expected.conf file, to explain what the file is about?